cmd/command.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+)
+
+type cmd struct {
+	name        string
+	command     func()
+	help        string
+	longHelp    string
+	subCommands map[string]*cmd
+}
+
+func (c *cmd) addSubCmds(subCmd ...*cmd) {
+	for _, sc := range subCmd {
+		c.subCommands[sc.name] = sc
+	}
+}
+
+// Suffice runner interface.
+func (c *cmd) run() {
+	c.command()
+}
+
+// Suffice helper interface.
+func (c cmd) printRegisteredCmds() {
+	b := helpHeaderBuilder("Usage of '%s':\n", c.name)
+
+	for n, sc := range c.subCommands {
+		b.WriteString(fmt.Sprintf("\t%-10s\t\t %s\n", n, sc.help))
+	}
+
+	fmt.Println(b.String())
+}
+
+func (c cmd) printHelp() {
+	b := helpHeaderBuilder("Usage of subcommand: '%s'\n", c.name)
```

Comment:
Other places we put the colon at the end of the prefix, for consistency, I'd recommend to use it here as well like this:

```suggestion
	b := helpHeaderBuilder("Usage of subcommand '%s':\n", c.name)
```

Reply:
Done